app/migrations/Version20170904153003.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Migrations;
+
+use AppBundle\DataFixtures\ORM\LoadSkillData;
+use Cocur\Slugify\Slugify;
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerAwareTrait;
+
+class Version20170904153003 extends AbstractMigration implements ContainerAwareInterface
+{
+    use ContainerAwareTrait;
+
+    public function up(Schema $schema)
+    {
+        foreach (LoadSkillData::SKILL_CI as $skill) {
+            $skillSlug = $this->container->get('sonata.core.slugify.cocur')->slugify($skill);
+            $existingSkill = $this->connection->executeQuery('SELECT id FROM skills WHERE slug LIKE :skillSlug', ['skillSlug' => $skillSlug]);
```

Comment:
If we use a LIKE statement here then we could have collisions.
e.g: The skill named 'Veille' will match the skill named 'Veille juridique'

Why don't we use a strict comparison ?

Reply:
In this case only 'Veille' will be found. `LIKE` without wildcard is the same as strict comparison, but I'm agree that the last is better.